src/middleware.ts

```diff
@@ -187,3 +188,19 @@ export const withJWTVerified =
 
     return handler(request);
   };
+
+export const withExistingRegistration =
+  (apl: APL): Middleware =>
```

Comment:
can we decide that we will use `{apl: APL}` object for named parameters? It's not only better from DX perspective (I know exlicitly what I pass there), but also will protect us from breaking changes and multiple arguments in the future (we can just extend object)